components/usage/pkg/controller/reconciler.go

```diff
@@ -57,10 +61,25 @@ func (u *UsageReconciler) Reconcile() error {
 		return err
 	}
 	log.WithField("usage_reconcile_status", status).Info("Reconcile completed.")
+
+	// For now, write the report to a temp directory so we can manually retrieve it
+	f, err := ioutil.TempFile("usage_report", fmt.Sprintf("%s-*", now.String()))
+	if err != nil {
+		return fmt.Errorf("failed to create temporary file: %w", err)
+	}
+	defer f.Close()
+
+	enc := json.NewEncoder(f)
+	err = enc.Encode(status.Report)
+	if err != nil {
+		return fmt.Errorf("failed to marshal report to JSON: %w", err)
+	}
+
```

Comment:
Should we write this to stdout instead? What's the expected workflow for getting hold of the report? With `kubectl exec`?

Reply:
Yep, for now it would be kubexec. It does also log it when it prints the status (which contains the report) but that may not be as easily retrievable as the file.